pkg/build/strategies/dockerfile/dockerfile.go

```diff
@@ -180,9 +176,9 @@ func (builder *Dockerfile) CreateDockerfile(config *api.Config) error {
 	if assembleProvided {
 		buffer.WriteString(fmt.Sprintf("RUN %s\n", sanitize(filepath.Join(scriptsDestDir, "assemble"))))
 	} else {
-		buffer.WriteString(fmt.Sprintf("# Assemble script sourced from builder image based on user input or image metadata."))
-		buffer.WriteString(fmt.Sprintf("# If this file does not exist in the image, the build will fail."))
-		buffer.WriteString(fmt.Sprintf("RUN %s\n", sanitize(filepath.Join(inImageScriptsDir, "assemble"))))
+		buffer.WriteString(fmt.Sprintf("# Assemble script sourced from builder image based on user input or image metadata.\n"))
+		buffer.WriteString(fmt.Sprintf("# If this file does not exist in the image, the build will fail.\n"))
+		buffer.WriteString(fmt.Sprintf("RUN %s\n", sanitize(filepath.Join(config.ImageScriptsPath, "assemble"))))
```

Comment:
Don't see where this is defaulted to `/usr/libexec/s2i` if the user does not explicitly set it ...  maybe `checkValidDirWithContents` does that?

Or should we do a len check on `config.ImageScriptsPath` and give `/usr/libexec/s2i` a go as a last resort?

Reply:
it's defaulted when the flag is setup:

`buildCmd.Flags().StringVar(&(cfg.ImageScriptsDir), "image-scripts-dir", "/usr/libexec/s2i", "Specify the absolute path of the directory containing the assemble and run scripts within the builder image")`